config/puma/production.rb

```diff
@@ -44,3 +44,10 @@
   dynflow = ::Rails.application.dynflow
   dynflow.initialize! unless dynflow.config.lazy_initialization
 end
+
+# Loading and initializing of all gettext languages takes about 100ms per language
+# in development environment and little less on production. Let's eager load languages
+# but only for production.
+on_worker_boot do
+  FastGettext.human_available_locales
+end
```

Comment:
On IRC I suggested to use `before_fork` so it can be shared between processes. That helps to reduce memory usage. Probably good to put that in the comment as well (if it actually works).

Reply:
Oh copy-paste error. Let me put that into the correct block.